.chloggen/3330-python-otel-logs-exporter.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
How is this enhancing the current code?

Reply:
The idea was to have one variable less to set for people wanting to export logs

Reply:
So before we didn't set OTEL_LOGS_EXPORTER, where were the logs sent then?

Reply:
They are not exported anywhere, docs tells users to set it
https://opentelemetry.io/docs/kubernetes/operator/automatic/#auto-instrumenting-python-logs

Docs I think is a bit outdated after 7d59d65f9406f69b67451f67a774c2ac74bc3bfb

Reply:
alright, so with this patch we are enabling log collection from python workloads. I think we should make this more clear in the changelog as it has implications e.g. on their collector config.

Reply:
We are not enabling log collection that is gated behind OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED, we are just removing the need to set the export protocol if one does so.

Reply:
@pavolloffay something like this? https://github.com/open-telemetry/opentelemetry-operator/pull/3330/files#r1791558054